RecoTauTag/HLTProducers/interface/L1PFJetsMatching.h

```diff
@@ -0,0 +1,38 @@
+#ifndef L1PFJetsMatching_H
```

Comment:
Hi, could you please clarify how to proceed with this? I looked at other examples in the HLTProducers and they were consistent with this line.